telco-hub/configuration/reference-crs/required/gitops/ztp-installation/app-project.yaml

```diff
@@ -3,8 +3,6 @@ kind: AppProject
 metadata:
   name: ztp-app-project
   namespace: openshift-gitops
-  annotations:
-    argocd.argoproj.io/sync-wave: "100"
```

Comment:
We need to keep all these sync-waves. If we need to update the ran reference first to ensure that they are kept when you do the copy, let's make that change too.

Reply:
Yes, fix on ran "first". The RAN reference is, however, in this same repo so we can fix both in this same PR :)

Reply:
I guess that RAN directory is just getting all the manifests directly from:
https://github.com/openshift-kni/cnf-features-deploy/tree/release-4.17/ztp/gitops-subscriptions/argocd/deployment
and there, the app-project.yaml does not have this content. So, we will create a divergence anyway.
I am not sure how this `sync-wave`, ` ImageClusterInstall` reached into here. I had an script to generate it directly from the ztp-site-generate, I guess we did the changes manually after executing the script.